apps/req.c

```diff
@@ -136,10 +136,10 @@ const OPTIONS req_options[] = {
      "Cert extension section (override value in config file)"},
     {"reqexts", OPT_REQEXTS, 's',
      "Request extension section (override value in config file)"},
-    {"precert", OPT_PRECERT, '-', "Add a poison extension (implies -new)"},
+    {"precert", OPT_PRECERT, '-', "Add a poison extension to generated cert (implies -new)"},
```

Comment:
the generated cert?

Reply:
Yes, I can add the 'the', which I had left out for brevity, but there is still room in the line (compared to other lines).

Reply:
Done